skrifa/src/outline/autohint/style.rs

```diff
@@ -434,13 +482,46 @@ mod tests {
             (96, Some(("Adlam", true))),
             (97, Some(("Adlam", false))),
         ];
+        check_styles(font_test_data::AUTOHINT_CMAP, ShaperMode::Nominal, expected);
+    }
+
+    #[test]
+    fn shaped_glyph_styles() {
+        // generated by printf debugging in FreeType
+        // (gid, Option<(script_name, is_non_base)>)
```

Comment:
What does `is_none_base` mean? That it's not from a basic char to glyph mapping?

Reply:
Got it thanks, not base as in non base character... ok, perhaps at a comment?

Reply:
Done... hopefully will clear up confusion for future readers :)